src/Components/ADT3DSceneBuilder/Internal/Behaviors/BehaviorsForm.tsx

```diff
@@ -401,26 +422,46 @@ const SceneBehaviorsForm: React.FC<IADT3DSceneBuilderBehaviorFormProps> = ({
     ]);
 
     const isFormValid = checkValidityMap(behaviorState.validityMap);
+    const isWidgetFormActive =
+        widgetFormInfo.mode === WidgetFormMode.CreateWidget ||
+        widgetFormInfo.mode === WidgetFormMode.EditWidget;
+    const isVisualRulesFormActive = visualRuleFormMode !== null;
     const theme = useTheme();
     const commonPanelStyles = getLeftPanelStyles(theme);
-    const commonFormStyles = getPanelFormStyles(theme, 168);
+    console.log('isPropertyDropdownEnabled: ', isPropertyTypeDropdownEnabled);
+    const formHeaderHeight = isVisualRulesFormActive
+        ? isPropertyTypeDropdownEnabled
```

Comment:
I don't like that this behavior form needs to know such a specific detail about a specific sub component. Is there a way to make the naming more generic or something?

Reply:
Why generic? If this is a very specific value maybe I should make even more explicit?